pom.xml

```diff
@@ -110,6 +110,7 @@
         <module>extensions-contrib/parquet-extensions</module>
         <module>extensions-contrib/statsd-emitter</module>
         <module>extensions-contrib/orc-extensions</module>
+        <module>extensions-contrib/thrift-extensions</module>
```

Comment:
while you are in this code, can you sort the extensions lexicographically by name?